recipe/meta.yaml

```diff
@@ -44,9 +44,15 @@ about:
   home: https://thrift.apache.org
   license: Apache-2.0
   license_family: APACHE
+  license_file: README.md
```

Comment:
The license_file is `LICENSE` https://github.com/conda-forge/thrift-feedstock/blob/304973c4a847582f313587a387238960e755ccb7/recipe/meta.yaml#L42 **not** `README.md`

Reply:
Yes, for the github package, that's correct. For pypi, the build failed because that file was not present.

@SC426 please also update the license URL.